### PR TITLE
Build grpc fat_cc_library on Windows

### PR DESCRIPTION
**Pull request by aherrmann-da**
- Modify `fat_cc_library` rule to support Windows.
  - Had to move `-lstdc++` to the end, so that when linked statically (as on Windows) the symbols are know to be required.
  - Had to add `-lws2_32` on Windows.
  - Had to adapt the compiler executable on Windows.
  - Choose to use the `cc_toolchain` `cc` executable on Linux as well, as it is only problematic on Darwin.
  - Use `static_library` instead of `pic_static_library` on Windows, as the former is not available.
- Add grpc `fat_cbits` to Windows CI. (Currently disabled)

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](./CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](./docs/source/support/release-notes.rst), if appropriate

### CODE REVIEW

File: bazel_tools/fat_cc_library.bzl [hunk 1]
```diff
@@ -1,44 +1,60 @@
...
     input_lib = ctx.attr.input_lib
     cc_info = input_lib[CcInfo]
-    pic_static_libs = []
+    static_libs = []
     # For now we assume that we have static PIC libs for all libs.
     # It should be possible to extend this but we do not have a need
     # for it so far and it would complicate things.
     for lib in cc_info.linking_context.libraries_to_link:
-        if not lib.pic_static_library:
-            fail("No PIC static lib for: " + lib)
```
Comment (aherrmann-da) [inline bazel_tools/fat_cc_library.bzl]: The `+ lib` caused a type error. `lib` is a Java object and can't be concatenated with a string. Also, it's string representation is not informative, so I've dropped it all together.

Comment (moritzkiefer-da) [reply bazel_tools/fat_cc_library.bzl]: Can we use something like lib.label? Some kind of hint as to what went wrong seems useful.

Comment (aherrmann-da) [reply bazel_tools/fat_cc_library.bzl]: Sadly there's no `.label` attribute. I've added the path to the dynamic library into the error message.

Comment (moritzkiefer-da) [reply bazel_tools/fat_cc_library.bzl]: :+1: That should be sufficient for debugging

**Pull request merged**